runtime-attach/build.gradle.kts

```
@@ -11,4 +11,15 @@ dependencies {
 
   // Used by byte-buddy but not brought in as a transitive dependency.
   compileOnly("com.google.code.findbugs:annotations")
+
+  testImplementation("io.opentelemetry.javaagent:opentelemetry-javaagent:1.13.1")
+  testImplementation("io.opentelemetry:opentelemetry-extension-annotations:1.14.0")
```

I don't think the version is needed here, it should get picked up from `io.opentelemetry:opentelemetry-bom`
```suggestion
  testImplementation("io.opentelemetry:opentelemetry-extension-annotations")
```